hazelcast/src/test/java/com/hazelcast/test/starter/HazelcastStarterUtils.java

```diff
@@ -16,21 +16,32 @@
 
 package com.hazelcast.test.starter;
 
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+
 import static org.junit.Assert.assertEquals;
 
-public class Utils {
-    public static final boolean DEBUG_ENABLED = false;
+@SuppressWarnings("WeakerAccess")
+public class HazelcastStarterUtils {
+
+    private static final boolean DEBUG_ENABLED = false;
+
+    private static final ILogger LOGGER = Logger.getLogger(HazelcastStarterUtils.class);
 
-    public static RuntimeException rethrow(Exception e) {
+    public static RuntimeException rethrowGuardianException(Exception e) {
         if (e instanceof RuntimeException) {
             throw (RuntimeException) e;
         }
         throw new GuardianException(e);
     }
 
+    public static boolean isDebugEnabled() {
```

Comment:
This tricks the static code analyzer to not falsely detect "dead code" in other classes.